cmds/webboot/webboot.go

```diff
@@ -207,6 +207,32 @@ func (d *DirOption) exec(uiEvents <-chan ui.Event) (menu.Entry, error) {
 	return menu.PromptMenuEntry("Distros", "Choose an option:", entries, uiEvents)
 }
 
+func mirrorMenu(entry menu.Entry, uiEvents <-chan ui.Event, link string) (url string, mirrorNameForTestPurposes string, err error) {
+	// Code for after the specific distro has been selected.
+	// Looks up the distro.
+	distro := supportedDistros[entry.Label()]
+	if len(distro.mirrors) > 0 {
+		// Make an array of type menu.Entry to store the mirrors of the
+		// particular distro selected. Then, display the mirror options.
+		entries := make([]menu.Entry, len(distro.mirrors))
+		for i := range entries {
+			entries[i] = &distro.mirrors[i]
+		}
+		entry, err = menu.PromptMenuEntry("Available Mirrors", "Choose an option:", entries, uiEvents)
+		if err != nil {
+			return "", "", err
+		}
+	}
+	// Iterate through the mirrors of the distro to select the appropriate link.
+	i := 0
```

Comment:
You don't need this line.

Reply:
I get an error if I delete that line. I need that line so that I can reference i outside of the for loop.